src/agent/src/signal.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2019-2020 Ant Financial
+// Copyright (c) 2020 Intel Corporation
```

Comment:
2021?

Reply:
I'm guessing the codes written in 2020 :)

Reply:
Yep. If the CI fails again, I'll update, but if it passes, *please* can we just merge it as this has got held up by all sorts of stuff (conflicts and rebases, wedged CI, linter issues, C19, aliens, unicorns, cosmic rays, *etc, etc* :)

Reply:
good (bad for you) news, jenkins-ci-debian-10 failed :sweat_smile: